constructors.js

```diff
@@ -60,51 +70,92 @@
  * @method  spendMana
  * @method  invoke
  */
+function Spellcaster(name, health, mana) {
+  this.name = name;
+  this.health = health;
+  this.mana = mana;
+  this.isAlive = true;
+}
+Spellcaster.prototype = Object.create(Spell.prototype);
```

Comment:
Spellcaster should not extend Spell. A spellcaster is not a more specific version of a spell!